build-system/compile/compile-wrappers.js

```diff
@@ -68,6 +61,7 @@ function extensionPayload(name, version, latest, isModule, loadPriority) {
   }
   // Use a numeric value instead of boolean. "m" stands for "module"
   const m = isModule ? 1 : 0;
+  const latest = latestVersions[name] === version;
```

Comment:
The intention with this helper is that if a publisher used `latest` at RTV 123, but 123 is outdated, then they really wanted `latest` at RTV 456. We couldn't just naively translate from the currently loaded version 0.1, because that was the latest at RTV 123. When we published RTV 456, the latest could have been 0.2.

But now that we're locking this down, I don't think we need this anymore.

Reply:
Sure, I'll revert the revert